src/Api/Value/Request/Message.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace KafkaRest\Api\Value\Request;
+
+use JMS\Serializer\Annotation as Serializer;
+
+final class Message
+{
+    /**
+     * @Serializer\Type("string")
+     * @var string|null
+     */
+    public $key;
```

Comment:
Properties order inverted

Reply:
Well the logic there is to go from "least concrete" field to the most identifying one. You can see that in the Response message - I wanted to keep the order the same. And constructor is inverted just because the key is optional.